project1/styles.css

```
@@ -0,0 +1,86 @@
+blockquote p {
+    font-family: 'Satisfy', cursive;
+    font-size: 30px;
+    line-height: 1;
+    margin-left: 150px;
+    margin-right: 150px;
+    text-align: center;
+}
+
+body {
+
+    color: white;
+}
+
+div {
+    font-size: 40px;
+    text-align: center;
+}
+
+h1 {
+    margin-bottom: 0;
+}
+
+a {
+    margin-top: 0
+}
+
+header {
+    color: navy;
+    font-family: 'Fjalla One', sans-serif;
+    font-size: 60px;
+    letter-spacing: 3px;
+
+}
+
+h1 {
+    margin-top: 50px;
+    margin-bottom: 10px;
+}
+
+p {
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 300;
+    line-height: 1.75;
+}
+
+section p {
+    font-size: 15px;
+    margin-bottom: 20px;
+    margin-left: 150px;
+    margin-right: 150px;
+    text-align: justify;
+}
+
+
+main {
+    background-color: rgb(14, 190, 204);
+    border: 20px gray;
```

This is missing a property that sets the `border-style` -- this will make your border show up 👍🏻
```suggestion
    border: 20px solid gray;
```

---

I kept wondering what I was missing, I didn't realize I needed the "solid"